common-docs/content/more-sources/repos/_index.md

```diff
@@ -0,0 +1,30 @@
++++
+title="Repos"
+description="This platform is tightly integrated with GitHub"
+layout="slimdex"
++++
+
+This platform is tightly integrated with GitHub. Really the _main_ content should always be project work in GitHub repositories, and this site should be a viewer for that content.
+
+You can pull a readme from any repository, or any subdir in any repo. We also set coursework assignments as issues in a relevant repo. Learners submit their work by creating a pull request to the project repo.
+
+They plan their work on GitHub projects, and track their progress on the issues. We code review via the GitHub PR interface, and we have some automation and unit testing on quizzes and exercises/problems.
+
+We always need to teach all our learners how to use Git and GitHub and modern development practices, so we do almost everything right on GitHub. Our volunteers are all on GitHub already and can throw together a repo in minutes, so this is a desire line design: we go to where the community is already.
```

Comment:
Do we want to say anything about affordances we've made for non-GitHub-familiar volunteers?

Reply:
Oh yes that's a good idea - talk about some cms options? I didn't want to go too deep in SSG lore but maybe here that's helpful